test/controllers/songs_controller_test.rb

```diff
@@ -1,19 +1,22 @@
-require 'test_helper'
+require "test_helper"
 
 class SongsControllerTest < ActionController::TestCase
-  # test "should get index" do
-  #   get :index
-  #   assert_response :success
-  # end
+  test "should get index" do
+    get :index
+    assert_response :success
+  end
 
-  # test "should get new" do
-  #   get :new
-  #   assert_response :success
-  # end
+  test "index should retrieve relevant search results" do
+    get :index, sSearch: "hand", format: :json, xhr: true
+    songs = JSON.parse(@response.body)["data"]
+    song_names = songs.map { |s| s["name"] }
+    assert_includes(song_names, songs(:hands_to_the_heaven)[:name])
```

Comment:
You can just do `assert_includes(songs, songs(:hands_to_the_heaven))` without going the roundabout way of comparing names, which is flaky anyway as multiple songs can have the same name.